plumgrid_playbooks/roles/plumgrid_director/tasks/main.yml

```diff
@@ -38,7 +34,7 @@
 - name: Login to PLUMgrid
   shell: >
     curl -H "Accept: application/json" -H "Content-Type: application/json" -k -X POST https://{{ pg_vip }}/0/login -d '{"userName":"{{ pg_username }}","password":"{{ pg_password }}"}' -c /tmp/cookie -i | grep HTTP | awk '{print $2}'
-  when: install_license == True and inventory_hostname == groups['shared-infra_hosts'][0]
+  when: install_license == True and inventory_hostname == groups['director_hosts'][0]
```

Comment:
shared-infra_hosts are required for OSA, since its the name of group containing infras.